dev-infrastructure/configurations/mvp-mgmt-cluster.bicepparam

```diff
@@ -0,0 +1,27 @@
+using '../templates/mgmt-cluster.bicep'
+
+param kubernetesVersion = '1.29.2'
+param istioVersion = 'asm-1-20'
+param vnetAddressPrefix = '10.132.0.0/14'
+param subnetPrefix = '10.132.8.0/21'
+param podSubnetPrefix = '10.132.64.0/18'
+param enablePrivateCluster = false
```

Comment:
do we need private cluster anymore?

Reply:
not to my knowledge

Reply:
99% no, but we've been holding until https://issues.redhat.com/browse/ARO-7842 completes to remove this